File: prototype/java/starcraft/src/main/java/lines/service/domain/MarineAction.java
package lines.service.domain;

import lines.model.UnitAction;

public class MarineAction implements UnitAction {
    @Override
    public void Harvest() {
        System.out.println("수확합니다.");
    }

    @Override
    public void Attack() {
        System.out.println("공격합니다.");
    }

    @Override
    public void Building() {
        System.out.println("건설합니다.");
    }
}

File: prototype/kotlin/starcraft/src/main/kotlin/lines/service/domain/MarineAction.kt
package lines.service.domain

import lines.model.UnitAction

class MarineAction : UnitAction {
    override fun Harvest() {
        print("수락합니다.")
    }

    override fun Attack() {
        print("공격합니다.")
    }

    override fun Building() {
        print("건설합니다.")
    }
}